sql/moz-fx-data-shared-prod/firefox_accounts_derived/nonprod_fxa_server_events_v1/metadata.yaml

```diff
@@ -0,0 +1,37 @@
+---
+friendly_name: FxA Server Events (nonprod)
+description: |
+  FxA server events extracted from accounts server stdout logs.
+
+  This new table is the direct result of FxA migration from AWS to GCP
+  (see: OPST-296 for more context).
+
+  Effective 2023-05-26 the events from the following servers land in this table
+  (existing data will remain in those tables):
+  - auth    (previous table: `nonprod_fxa_auth_events_v1`)
+  - content (previous table: `nonprod_fxa_content_events_v1`)
+
+  Payment server events will continue landing inside: `nonprod_fxa_stdout_events_v1`
```

Comment:
cc @sean-rose

Reply:
regarding the renaming to `auth_content_events` this would also not be desirable as in the future the payments events might also end up landing in this table and once again we end up with naming confusion.

Alternative here could be to have our ETL include the events from the existing `payments` (stdout) table and the data coming from multiple tables being an implementation detail. This is what it would look like: https://docs.google.com/document/d/1fe6aUXHPU3P4ZZItDB5Wc2r8Y6I_pI9m1BVcGQjIqs0/edit?disco=AAAAzAOKPg4

Reply:
Given that design vision I'd be inclined to include payments server events in this ETL now (i.e. option 2).  You'll just need to be careful with having a hard cutoff (as opposed to the overlapping cutoff for the other logs that transitioned) in both the`server_events` ETL for when to start including existing `stdout` logs and in `nonprod_fxa_all_events` for when to stop including logs from the existing `stdout` ETL (otherwise `nonprod_fxa_all_events` will select duplicate `stdout` logs).

Reply:
> You'll just need to be careful with having a hard cutoff (as opposed to the overlapping cutoff for the other logs that transitioned) in both theserver_events ETL for when to start including existing stdout logs and in nonprod_fxa_all_events for when to stop including logs from the existing stdout ETL (otherwise nonprod_fxa_all_events will select duplicate stdout logs).

IMO this is too complex, it's easier to reason about what's going on in these logs if there's 1-1 relationship between these tables and their sources.
I'd rather opt for either keeping this query as is (I agree that naming is not ideal and am open to suggestions) or separate `auth` and `content` (although this breaks 1-1 relationship mentioned above, but is simpler to reason about in case someone's e.g. debugging some missing data).

Reply:
Yeah, there's no ideal solution here.

If there's a definite plan to switch to having payments events also land in `moz-fx-fxa-nonprod.gke_fxa_stage_log.stdout` (and similarly for production) then I think it'd make sense to keep this as `server_events`.  Otherwise I'd personally be inclined to have separate `auth` and `content` v2 ETLs (i.e. option 3).

Reply:
> If there's a definite plan to switch to having payments events also land in `moz-fx-fxa-nonprod.gke_fxa_stage_log.stdout` (and similarly for production)

No concrete plan exists yet. It will probably happen in the future, next year at the earliest.